app/src/main/java/com/atom/android/bookshop/ui/account/AccountPresenter.kt

```diff
@@ -0,0 +1,56 @@
+package com.atom.android.bookshop.ui.account
+
+import android.content.Context
+import com.atom.android.bookshop.data.model.User
+import com.atom.android.bookshop.data.repository.AccountRepository
+import com.atom.android.bookshop.data.source.remote.IRequestCallback
+import com.atom.android.bookshop.data.source.remote.ResponseObject
+import com.atom.android.bookshop.utils.SharedPreferenceUtils
+import com.atom.android.bookshop.utils.destroyToken
+import com.atom.android.bookshop.utils.getTokenLogin
+
+
+class AccountPresenter(
+    private val repository: AccountRepository,
+    private val accountView: AccountContract.View
+) : AccountContract.Presenter {
+
+    override fun getUser(context: Context?) {
+        val token = SharedPreferenceUtils.getInstance(context)?.getTokenLogin()
+        repository.getUser(token, object : IRequestCallback<ResponseObject<User>> {
+            override fun onSuccess(responseObject: ResponseObject<User>) {
+                accountView.getUserSuccess(responseObject.data as User)
+            }
+
+            override fun onFailed(message: String?) {
+                accountView.getUserFailed(message)
+            }
+
+        })
+    }
+
+    override fun destroyToken(context: Context?) {
+        SharedPreferenceUtils.getInstance(context)?.destroyToken()
+    }
+
+
+    override fun onStart() {
+        // late impl
```

Comment:
```suggestion
        // TODO implement later
```
Sửa hết cho mấy chỗ khác em nhé!

vì sao cmt như này em đọc ở đây: https://www.jetbrains.com/help/idea/using-todo.html